examples/kms.js

```diff
@@ -12,7 +12,22 @@ const KMS = new KMSClient(awsConfig)
 const KeyId = 'alias/TestKey'
 
 export default function () {
-    //Run GenerateDataKey call on the key, with the default 32 byte size 
-    KMS.GenerateDataKey(KeyId)
-}
+    // Currently, the keys need to be created before hand
+
+    // First let's list the keys we have available
+    const keys = KMS.listKeys();
+    if (!keys.length == 0) {
```

Comment:
Shouldn't it be

```suggestion
    if (keys.length == 0) {
```

instead?

Reply:
yeah, that's what i get for just eyeballing code and not running it...

Reply:
updated.